nchs_mortality/delphi_nchs_mortality/archive_diffs.py

```diff
@@ -32,12 +32,15 @@ def arch_diffs(params, daily_arch_diff, logger):
     # Weekly run of archive utility on Monday and Thursday
     # - Does not upload to S3, that is handled by daily run of archive utility
     # - Exports issues into receiving for the API
+    n = 0
     if datetime.today().weekday() == (0 or 3):
```

Comment:
add an `else` block to this so that we can see from the logs that if we didn't publish that day, we did so on purpose